modules/web-console/frontend/app/modules/agent/AgentManager.service.ts

```diff
@@ -165,9 +169,6 @@ class ConnectionState {
     }
 
     disconnect() {
-        if (this.cluster)
```

Comment:
Why did you remove this?

Reply:
not found usage of this.cluster.disconnect